text/chapter3.md

```diff
@@ -678,22 +675,25 @@ Wrapping an infix operator in parentheses lets you use it as a prefix function:
 11
 ```
 
-Alternatively, operators can be partially applied by surrounding them with parentheses and using `_` as an operand in an [operator section](https://github.com/purescript/documentation/blob/master/language/Syntax.md#operator-sections):
+Alternatively, operators can be partially applied by surrounding the expression with parentheses and using `_` as an operand in an [operator section](https://github.com/purescript/documentation/blob/master/language/Syntax.md#operator-sections). You can think of this as a more convenient way to create simple anonymous functions (although in the below example, we're then binding that anonymous function to a name, so it's not so anonymous anymore):
 
 ```text
 > add3 = (3 + _)
 > add3 2
 5
 ```
 
-To summarize, the following are equivalent definitions of a function  that adds `5` to its argument:
+To summarize, the following are equivalent definitions of a function that adds `5` to its argument:
 
 ```haskell
 add5 x = 5 + x
 add5 x = add 5 x
 add5 x = (+) 5 x
 add5 x = 5 `add` x
 add5   = add 5
+add5   = \x -> 5 + x
+add5   = (5 + _)
+add5 x = 5 `(+)` x  -- Yo Dawg, I herd you like infix, so we put infix in your infix!
```

Comment:
Too much?

Was hoping that this would work too:
```purs
add5 x = (`add`) 5 x
```

Reply:
My feeling is that book's overall tone is a bit drier, so I don't know how well the comment on the last line fits.

Personally, I like it, and earlier I had considered changing your "john" "peggy" "ned" example to "homer" "apu" "ned", but decided against it, for consistency.